mautic/mautic
# Fixed issue that prevented cloning of standalone forms

**Description**

This PR fixes issue where cloning standalone forms will fail when saving due to the alias being empty.

**Testing**

1. Create a form
2. Clone the form and select Standalone then try to save. It will uh oh in prod or give SQL error in dev.
3. Apply the PR then repeat and this time it should work.
4. Test creating new form, editing form, and cloning forms to ensure all work.
